### DISCUSSION TITLE
Custom Footer

### DISCUSSION THREAD

**Discussion by MuongKimhong** [Textualize/textual#3923] [Q&A]
https://textual.textualize.io/widgets/footer/#textual.widgets

I want to create a custom Footer that I can change the description or interact with like text input within Footer, but I don't see any attributes or any available sources that can help me.

**Comment by TomJGooding** [comment]
Sorry I'm not sure what you mean by "interact with like text input", could you elaborate? I assume by "change the description" you mean the bindings?

**Reply by MuongKimhong** [reply]
It's alright! It's a bit hard to explain but I think I can just create a custom widget instead. Thanks you.
